src/ansi-c/expr2c.cpp

```diff
@@ -2082,7 +2082,7 @@ std::string expr2ct::convert_union(
   if(src.operands().size()!=1)
     return convert_norep(src, precedence);
 
-  std::string tmp=convert(src.op0());
+  std::string tmp = convert(to_union_expr(src).op());
```

Comment:
Hmm, could we actually get rid of `tmp` here and just inline the `convert` call below, where `tmp` is currently used? (It would be nice if over time we could get rid of any uses of the non-descriptive name `tmp`.)